### PR TITLE
5 view individual article

### PR DESCRIPTION
**Pull request by Isaac-Sturtridge**
CORE: View an individual article

Current display on the individual article page I am happy with for this ticket, added a shell file for articleList that will be replaced with the one on branch 4 (plus Link functionality), here solely to display the links to individual articles on the main page.

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import {useParams} from "react-router-dom"
+import {getSingleArticle} from "../utils/axios"
+
+const Article = () => {
+  const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true)
+  const { article_id } = useParams();
```
Comment (2202Hannah) [inline src/components/Article.jsx:8]: nice use of useParams() 👍

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      .then((response) => {
+        return response.data;
+      })
+      .then((data) => {
+        setArticle(data.article);
+        setIsLoading(false)
+      });
+  }, []);
+
+  if(isLoading) {
```
Comment (2202Hannah) [inline src/components/Article.jsx:21]: This loading state and conditional rendering is great for the user experience

File: src/components/ArticleCard.jsx [hunk 3]
```diff
@@ -1,9 +1,12 @@
+import {Link} from 'react-router-dom';
+
 const ArticleCard = ({article}) => {
   return (
     // very basic layout, to be updated later
-    <article className="article_card">
+    <article className="article_card"> 
+    <Link to={`/articles/${article.article_id}`}>
```
Comment (2202Hannah) [inline src/components/ArticleCard.jsx:7]: great job linking to the single article page here :)

**Review by 2202Hannah — APPROVED** [review]
Really nice work on this ticket Isaac. 

You are probably aware of this already, but I had to navigate to the articles page via the URL as there was no button to take me there on your landing page. I would suggest adding this functionality in soon if you haven't already on another branch.

**Comment by Isaac-Sturtridge** [comment]
> Really nice work on this ticket Isaac.
> 
> You are probably aware of this already, but I had to navigate to the articles page via the URL as there was no button to take me there on your landing page. I would suggest adding this functionality in soon if you haven't already on another branch.

Oh yes, at the moment I am treating '/articles' like the landing page (to be edited later on), so I had better put in some redirection so that it does go there for now - or indeed put in a basic landing page that can link to articles, and indeed as I type it out the latter may be easier to quickly put in at this moment.

**Pull request merged**